src/utils.cpp

```diff
@@ -159,7 +157,7 @@ OutputVector pre_translate_ragged_string_tensor_input(ov::Output<ov::Node> input
 
 ov::Output<ov::Node> post_translate_string_tensor_output(const OutputVector& outputs) {
     FRONT_END_GENERAL_CHECK(outputs.size() == 3, "Expected 3 tensors in decomposed string tensor representation");
-    return std::make_shared<StringTensorPack>(outputs, "begins_ends");
+    return std::make_shared<StringTensorPack>(outputs[0], outputs[1], outputs[2]);
```

Comment:
please add explicit namespace from which you use `StringTensorPack`. We expect opset one.
For other lines, the same comment

Reply:
done